x-pack/plugins/timelines/kibana.json

```diff
@@ -5,6 +5,6 @@
   "configPath": ["xpack", "timelines"],
   "server": true,
   "ui": true,
-  "requiredPlugins": [],
+  "requiredPlugins": ["data", "dataEnhanced", "kibanaReact", "kibanaUtils"],
```

Comment:
consider adding `owner.name`, `owner.githubTeam`, and `description` to this `kibana.json`

example:

```
"owner": {
    "name": "App Services",
    "githubTeam": "kibana-app-services"
  },
  "description": "This plugin adds xyz functionality and exposes abc"
}
```